### DISCUSSION TITLE
Error due to `ggsave(limitsize=TRUE)` - how to work around this?

### DISCUSSION THREAD

**Discussion by gedkitchener-CT** [facebookexperimental/Robyn#316] [Q&A]
Hiya, I have set a high-ish number of `context_vars` - as such the prophet decomposition makes many plots (~20), which is making the plot height exceed 50 inches, and makes my script error out.

This is frustrating. The model works just fine, but because of this non-vital aspect (plotting), no output gets written at all! It feels like baking a beautiful cake, but because my camera can't fit the whole cake in shot, the cake just vanishes! "Oi, where's my cake gone!"

I've been killing myself all day trying to find a solution. Admittedly, I am new to R, so I'm probably missing something obvious.

I'm fairly certain it's down to the `limitsize=TRUE` default argument in `ggsave()`.

So please, can someone tell me how to change the default in `ggsave()` to be `limitsize=FALSE`. I'm quietly confident this will solve my problems.
* do I have to edit source code in `ggplot2`
* can I just redefine the function in my main script after `library(Robyn)`?
* etc.

**Comment by laresbernardo** [comment]
Hi @gedkitchener-CT 
Thanks for reporting this issue and I'm sorry you're not able to export your results. If you're familiar with cloning repos a making changes, feel free to do so. If you think your solution would help others as well, you can submit a pull request to merge it with our package. Said that I'll add this ticket to my backlog on potential improvements to be implemented soon.
Do try adding `limitsize = FALSE` in your clone and let us know how it goes

**Reply by melanie-wander** [reply]
I have a similar issue, but with the latest version I pulled 2 weeks ago.  I was the some plots and csv's, but none of the onepager plots out which recommended me to add  'limitsize=FALSE' in the run_robyn() function. I still get an error message however, it does at least give me the csvs and other plots.

**Reply by gedkitchener-CT** [reply]
Not sure if this helps, but after I manually added `limitsize=FALSE` in the `run_robyn()` function, I then ran `devtools::load_all()` which I guess is akin to 'sourcing' a bash file to propagate any changes through to the environment.

Anyway, you may have known to do that already, but I didn't. And it was a vital last step! So I'm putting it here for reference in case another R noob comes along.

**Comment by gufengzhou** [comment]
This should be fixed now with the new commit. please reinstall and test. https://github.com/facebookexperimental/Robyn/commit/eb0dc8efcf9bac699a42742bb269a43e7c123570

**Reply by gedkitchener-CT** [reply]
Thanks for the speedy fix
